src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx

```diff
@@ -298,8 +300,15 @@ const EnrolmentForm: React.FC<Props> = ({
               </FormGroup>
 
               <div className={styles.submitButtonWrapper}>
-                <Button type="submit">
-                  {t('enrolment:enrolmentForm.buttonSubmit')}
+                <Button
+                  type="submit"
+                  className={styles[enquiry ? 'enquiryButton' : 'enrolButton']}
+                >
+                  {t(
+                    `enrolment:enrolmentForm.${
+                      enquiry ? 'buttonSubmitEnquiry' : 'buttonSubmit'
+                    }`
+                  )}
```

Comment:
I would prefer:

```
 <Button
                  type="submit"
                  className={
                    enquiry ? styles.enquiryButton : styles.enrolButton
                  }
                >
                  {enquiry
                    ? t('enrolment:enrolmentForm.buttonSubmitEnquiry')
                    : t('enrolment:enrolmentForm.buttonSubmit')}
                </Button>
```

For readability reasons

Reply:
To me feels like those ternaries inside brackets needs little bit more work to understand. But not a must change, just my opinion

Reply:
fixed